DependencyInjection/SonataMediaExtension.php

```diff
@@ -428,7 +427,6 @@ public function configureFilesystemAdapter(ContainerBuilder $container, array $c
                     'version' => $config['filesystem']['s3']['version'],
                 ))
             ;
-
```

Comment:
This commit should be fixed up with the previous one : we don't want to see in the history your first try at formatting this.